selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -1433,27 +1426,16 @@ private void waitTextInJavaDoc(String expectedText) {
    *
    * <p>Note! {@link SeleniumWebDriver} should be switched to the javadoc frame.
    *
-   * <p>Please use {@link CodenvyEditor#waitAvailabilityAndSwitchToJavaDocFrame()} method for
-   * switching to javadoc frame.
-   *
    * @param expectedText text which should be present in javadoc body
    * @return true - if {@code expectedText} is present in javadoc body, false - if not
    */
   public boolean waitAndCheckTextPresenceInJavaDoc(String expectedText) {
     return seleniumWebDriverHelper
-        .waitVisibility(By.tagName("body"))
+        .waitVisibility(By.xpath(JAVA_DOC_POPUP))
```

Comment:
I guess this is a bugfix?

Reply:
Yes, before the quick documentation opened javadoc in an iframe but now quick documentation uses the built in hover which isn't an iframe so I had to change this

Reply:
ack